app/resources/documentation/manual/shortcuts.js

```diff
@@ -734,3 +734,5 @@ actions["Controls/Camera/Move forward"] = { key:87, modifiers: [] };
 actions["Controls/Camera/Move backward"] = { key:83, modifiers: [] };
 actions["Controls/Camera/Move left"] = { key:65, modifiers: [] };
 actions["Controls/Camera/Move right"] = { key:68, modifiers: [] };
+actions["Controls/Camera/Move up"] = { key:81, modifiers: [] };
```

Comment:
Hmm. Well, I do see code in getActionJSTable that appears like it needs to be modified to include those keys. However, doing some tests with TrenchBroom it doesn't seem like the JS file gets regenerated when I edit the preferences (which was my first guess on when that would be triggered), so I can't actually test that change.

I can experiment more later; out of time tonight tho. Or feel free to close this PR if you'd rather have me just file an issue.

Reply:
To regenerate shortcuts.js you need to launch a debug build, go to "Debug->Copy Javascript Shortcut Map" in the menu and it'll copy the new contents of shortcuts.js to the clipboard.. so it's sort of half-automated.

Reply:
There is something wrong here. I'm pretty sure that it's because you are not on macOS, which is the only OS I have used to generate that file so far. It looks like your file is missing the keycode name for the Cmd key, which other OSs don't have, obviously. Please just discard all changes to the file in this PR and I'll then regenerate the file on my mac and add it here.